projects/NXP/devices/iMX6/options

```diff
@@ -17,7 +17,7 @@
     KERNEL_TARGET="zImage"
 
   # kernel serial console
-    EXTRA_CMDLINE="console=ttymxc0,115200 console=tty0"
+    EXTRA_CMDLINE="console=ttymxc0,115200 console=tty0 video=HDMI-A-1:1920x1080@60"
```

Comment:
why is that needed? imx6 is capable of autodetecting native screen resolution.

Reply:
Yes please remove this change. You can add this to your own device if you need this.

Reply:
I get sometimes a resolution of 1024x768, sometimes 1920x1080.
Sometimes I even get 1024x768 on a TV with 1920x1080 in kodi.

![IMG_20201114_023138_sz](https://user-images.githubusercontent.com/40670278/99132720-603ba100-2628-11eb-9c69-120f73654744.jpg)

But perhaps this is due to the problems of reading EDID in my Wandboard.
I should probably remove this and investigate the issue better.

Reply:
I experienced similar issue on Allwinner images. Workaround is to disable display driver in U-Boot, at least in my case. You can modify U-Boot config via bootloader/config file.